Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -43,6 +44,12 @@ namespace firebase {
 namespace firestore {
 namespace testutil {
 
+/**
+ * A string sentinel that can be used with PatchMutation() to mark a field for
+ * deletion.
+ */
+constexpr const char* kDeleteSentinel = "<DELETE>";
```

Comment:
Isn't this already defined somewhere not specific to test? Or actually you want to test the sentinel is the same?

Reply:
No, it only seems to appear in the testutil class (at least for android, objc, and now c++)